app/models/manageiq/providers/nsxt/nsxt_client.rb

```diff
@@ -15,64 +15,68 @@ def initialize(server, user, password, verify_ssl = false)
   end
 
   def get_tier_1(id)
-    get("infra/tier-1s/#{id}")
+    get("tier-1s/#{id}")
   end
 
   def get_tier_1s
-    list('infra/tier-1s')
+    list('tier-1s')
   end
 
   def get_segment(id)
-    get("infra/segments/#{id}")
+    get("segments/#{id}")
   end
 
   def get_segments
-    list('infra/segments')
+    list('segments')
   end
 
   def get_security_policy(id)
-    get("infra/domains/default/security-policies/#{id}")
+    get("domains/default/security-policies/#{id}")
   end
 
   def get_security_policies
-    list('infra/domains/default/security-policies')
+    list('domains/default/security-policies')
   end
 
   def get_security_policy_rules(id)
-    list("infra/domains/default/security-policies/#{id}/rules")
+    list("domains/default/security-policies/#{id}/rules")
   end
 
   def get_service
-    get("infra/services/#{id}")
+    get("services/#{id}")
   end
 
   def get_services
-    list('infra/services')
+    list('services')
   end
 
   def get_group(id)
-    get("infra/domains/default/groups/#{id}")
+    get("domains/default/groups/#{id}")
   end
 
   def get_groups
-    list('infra/domains/default/groups')
-  end
-
-  def get_group_members(id)
-    list("infra/domains/default/groups/#{id}/members/virtual-machines")
+    list('domains/default/groups')
   end
 
   private
 
   def list(url)
     json = get(url)
-    return json['results'] || []
+    json.nil? || json['results'].nil? ? [] : json['results']
```

Comment:
This can be simplified as `json&.dig("results") || []`

Reply:
Fixed